### PR TITLE
FIX: Update Travis

### LINKED ISSUE #233: BUG: python3 compatibility
File "C:\Anaconda3\lib\site-packages\expyfun-2.0.0.dev0_unknown-py3.4.egg\expyfun\_experiment_controller.py", line 1633, in _get_items
    for key, val in d.iteritems():
AttributeError: 'dict' object has no attribute 'iteritems'

The method iteritems has changed to iter in python3.

### PR DESCRIPTION
**Pull request by Eric89GXL**
Closes #233

### CODE REVIEW

**Comment by Eric89GXL** [comment]
Ready for review/merge @rkmaddox @drammock. @mirbagheri this should fix the Py3k issue. If you find others feel free to open an issue or even better a PR for a fix.

**Comment by Eric89GXL** [comment]
@drammock feel free to review and merge this one if you're happy

File: .travis.yml [hunk 1]
```diff
@@ -40,7 +40,7 @@ before_install:
     - if [ "${TEST}" == "extra" ]; then
         travis_retry pip install -q flake8;
       else
-        travis_retry pip install -q nose-timer http://pyglet.googlecode.com/archive/tip.zip;
+        travis_retry pip install -q nose-timer http://bitbucket.org/pyglet/pyglet/get/tip.zip;
```
Comment (drammock) [inline .travis.yml]: I thought we switched to loading pyglet from a local copy, so we wouldn't get bitten by their backwards-incompatible changes in their perpetual alpha stage?  Did I dream that?

Comment (Eric89GXL) [inline .travis.yml]: I think we talked about it, but never did it. Maybe we won't need to...?

**Pull request merged**